content/en/05-reference/01-admin/09-user-mgmt.md

```diff
@@ -0,0 +1,109 @@
+
+3. Click the “+ Invite Users” button in the top right corner. This will pop open a modal with a generated invite link for your organization.
+
+<svg title='' src='admin/user-mgmt/invite-link.png'/>
+
+4. Copy the link and send it to your teammates.
+
+### Enable Approvals for New Users
+
+Combine Invite Links with User Approvals for greater control over who joins your organization. If your organization enables User Approvals, every user must be manually approved after they join your organization before they are able use Pixie.
+
+1. Open the [Live UI](/using-pixie/using-live-ui) Admin page (`/admin`).
+2. Select the Org Settings tab (`/admin/org`).
+3. Click “Enable” on the approval setting to enable Approvals. Note that if the button says “Disable”, this means User Approvals has already been enabled for your organization.
+
+<svg title='' src='admin/user-mgmt/enable-approvals.png'/>
+
+4. Manually approve new users under the Users tab (`/admin/org`).
+
+>> Once User Approvals are enabled, any new users that join your org will join in a disabled state.
+
+<svg title='' src='admin/user-mgmt/user-unapproved.png'/>
+
+>> Click “Approve” on the appropriate user to allow that user complete access to your organization within Pixie.
+
+<svg title='' src='admin/user-mgmt/user-approved.png'/>
+
+### Disable Invite Links
+
+Invite links are valid for 7 days from the time of creation and will expire automatically at the end of the 7 day period. If you would like to invalidate invite links before the 7 day period is over, you can do so on the Orgs page.
+
+<Alert variant="outlined" severity="warning">When you reset invite links, ALL existing invite links for your organization will be invalidated.</Alert>
+
+1. Open the [Live UI](/using-pixie/using-live-ui) Admin page (`/admin`).
+2. Select the Org Settings tab (`/admin/org`).
+
+<svg title='' src='admin/user-mgmt/reset-invite-links.png'/>
+
+3. Click the red "Reset Invite Links” button. This will pop open a modal to confirm that you would like to reset all existing invite links for your org.
+
+<svg title='' src='admin/user-mgmt/reset-invite-links-confirmation.png'/>
+
+4. Click “Reset Invite Links” in the modal to confirm and invalidate all existing invite links.
+
+## Remove Users
+
+1. Open the [Live UI](/using-pixie/using-live-ui) Admin page (`/admin`).
+2. Select the Users tab (`/admin/org`).
+3. Click the "Remove" button next to the desired user.
+
+<svg title='' src='admin/user-mgmt/users-tab.png'/>
+
+4. Accept the prompt confirming the removal.
+
+<svg title='' src='admin/user-mgmt/user-removal-prompt.png'/>
+
+Once a user is removed, they will be automatically logged out from any active UI sessions and will be unable to query Pixie resources.
+
+If you mistakenly remove a user and need to add the user back, you can create a new invite link and send it to the user. The user will be able to use their existing credentials to log in and join your org.
```

Comment:
should we drop this? because the general caveat of GSuite user vs not also applies here and people are liable to read bits of this page without reading other sections.